### DISCUSSION TITLE
signal Missed xdrip with oop2

### DISCUSSION THREAD

**Discussion by xema** [Navid200/xDrip#127] [Libre]
Hi I have been using xdrip with the oop 2 algorithm and libre 2 for several months. But for a couple of hours it has stopped working. In xdrip I have the error of signal missed and I cannot reconnect.

I've done everything I can think of, forced to close the apps, reboot the phone, cleared the xdrip settings and reconfigured it, tweak the bluetooht, rebooted the collector, etc.

however the patch is working correctly since from the Librelink app I receive the data correctly.

can anybody help me? Thanks

**Comment by xema** [comment]
For more information. I have a Xiaomi T10 Lite. And when pass NFC in Xdrip i have a status OK message

and the following messages appear in the app's error log
![Screenshot_2021-11-30-00-54-57-296_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/1434048/143961397-24a85032-9fb5-4c02-a521-c5f7f9cda5f9.jpg)
![Screenshot_2021-11-30-00-54-40-525_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/1434048/143961409-ddeff849-537f-49c5-b91d-a2ddc66a1749.jpg

:

**Comment by Navid200** [comment]
Are you still experiencing this problem?
